woocommerce/woocommerce-android
# Prologue survey experiment

This PR adds the prologue survey experiment logic. I've been waiting for a final copy update from marketing but there are some changes that are useful in the future experiments, so we can just merge it now and update the copy later. The A/B test is not running so the users will still see the original flow.

![image](https://user-images.githubusercontent.com/1522856/182163174-1f334dd3-3d3f-4700-be4c-28893d009b23.png)

**To test:**
1. Clear the app data
2. Start the app and click through the prologue
3. After clicking on Next button on the last prologue screen, notice the login screen is displayed (default behavior)
4. Change the `prologue_variant` value in `res/xml/remote_config_values.xml` to `survey`
5. Clear the app data again, run the app and click through the prologue
6. Notice the after finishing the prologue, the survey screen is displayed
7. Notice that after skipping/closing the survey screen, the login screen is displayed
